mokapot/config.py

```diff
@@ -252,6 +252,12 @@ def _parser():
         help=("Save the models learned by mokapot as pickled Python objects."),
     )
 
+    parser.add_argument(
+        "--init_weights",
+        type=str,
+        help=("Use saved models and skip training."),
+    )
```

Comment:
Although this matches Percolator, I would change the name of the argument to something like `--saved_models` and set it to be of length 1 or more, so that the user can specify all of the model files individually.